virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -102,6 +102,31 @@ def get_non_root_disk_name(session):
         return name.strip(), mpoint.strip()
 
 
+def get_disk_by_serial(session, serial):
```

Comment:
We already have same function in utils_disk.py.

`def get_disk_by_serial(serial_str, session=None):`

Reply:
Thanks for the pointer. I'll close this one.